Lib/typing.py

```diff
@@ -251,7 +252,7 @@ def _collect_parameters(args):
     """Collect all type variables and parameter specifications in args
     in order of first appearance (lexicographic order). For example::
 
-        _collect_parameters((T, Callable[P, T])) == (T, P)
+        assert _collect_parameters((T, Callable[P, T])) == (T, P)
```

Comment:
Guido mentioned in #101827 that he didn't like this style of code example, where it's not really a REPL example, but also isn't something you'd ever put in an executable `.py` file either. I agree that using an explicit `assert` is better for this kind of code example.